web/crux/prisma/schema.prisma

```diff
@@ -525,3 +528,34 @@ model Storage {
 
   @@unique([name, teamId])
 }
+
+model ConfigBundle {
+  id          String   @id @default(uuid()) @db.Uuid
+  name        String
+  description String?
+  data        Json
+  createdAt   DateTime @default(now()) @db.Timestamptz(6)
+  createdBy   String   @db.Uuid
+  updatedAt   DateTime @updatedAt @db.Timestamptz(6)
+  updatedBy   String?  @db.Uuid
+
+  team   Team   @relation(fields: [teamId], references: [id], onDelete: Cascade)
+  teamId String @db.Uuid
+
+  deployments ConfigBundleOnDeployments[]
+
+  @@unique([name, teamId])
+}
+
+model ConfigBundleOnDeployments {
+  deployment   Deployment @relation(fields: [deploymentId], references: [id], onDelete: Cascade)
+  deploymentId String     @db.Uuid
+
+  configBundle   ConfigBundle @relation(fields: [configBundleId], references: [id])
```

Comment:
Let's check if set this  to cascade, then we can probably get rid of the team relation.

Reply:
Seems to be working with cascade without teamId :)